content/techniques/hot-reload.md

```diff
@@ -70,20 +151,20 @@ async function bootstrap() {
 bootstrap();
 ```
 
-And that's all. To simplify execution process, add those two lines into your `scripts` inside `package.json` file.
+To simplify the execution process, add two scripts to your `package.json` file.
 
 ```json
-"start": "node dist/server",
 "webpack": "webpack --config webpack.config.js"
+"start": "node dist/server",
 ```
 
-Now simply open your command line and run below command:
+Now simply open your command line and run the following command:
 
 ```bash
 $ npm run webpack
 ```
 
-Once webpack started to **watch files**, run another command in the another command line window:
+Once webpack started **watching files**, run another command in a separate command line window:
```

Comment:
Once webpack has started <- add *has*
run the following command <- replace *another* with *the following*